environment/credentials/types.go

```diff
@@ -96,6 +96,12 @@ type NutanixPrismEndpoint struct {
 	// +kubebuilder:default=false
 	// +optional
 	Insecure bool `json:"insecure"`
+	// AdditionalTrustBundle is a PEM encoded x509 cert for the RootCA that was used to create the certificate
+	// for a Prism Central that uses certificates that were issued by a non-publicly trusted RootCA. The trust
+	// bundle is added to the cert pool used to authenticate the TLS connection to the Prism Central.
+	// +kubebuilder:validation:Type=string
+	// +optional
+	AdditionalTrustBundle string `json:"additionalTrustBundle,omitempty"`
```

Comment:
Can we change this from string to a struct similar to NutanixCredentialReference? This will allow us to add additional data sources for the trust bundle. An example could be a ConfigMap

Reply:
updated the type; PTAL